### ISSUE TITLE
A forward slash '/' is lost

### ISSUE THREAD

**Issue by helsonxiao** [dadoonet/fscrawler#1160]
**Describe the bug**

A forward slash '/' is lost in windows shared folder document.

Here is a search result for index named like `*_file`, unix style path and windows path are correct but not windows shared folder. It becomes `/servername/someplace` and virtual path is also wrong.
![image](https://user-images.githubusercontent.com/26411634/120878461-327f6f00-c5ef-11eb-992e-c94110614329.png)

**Job Settings**

```yml
name: "windows_shared_file"

elasticsearch:
  nodes:
  - url: "http://127.0.0.1:9200"

fs:
  # Windows Shared Folder
  url: "//DESKTOP-666/test"

server:
  hostname: "192.168.18.207"
  port: 22
  username: "helsonxiao"
  password: "123456"
  protocol: "ssh"
```

**Expected behavior**

File path can be the same format as they are stored in `job_name_folder` index. I'm using this path to download file from several indexes(linux/macos/windows/windows shared folder), but I can't tell if it's a windows shared folder from current document.
![image](https://user-images.githubusercontent.com/26411634/120878602-32cc3a00-c5f0-11eb-8740-c72c36834ecf.png)

**Versions:**
 - OS: MacOS
 - Version: 2.7(running in docker container)

**Comment by helsonxiao** [comment]
Fixed in this [commit](https://github.com/waterstone-company/fscrawler/commit/8d32259e6c419c8c4af40af326be79e7ad1fcc76#diff-27aabc48cd80adbdd85f89c53ff70563163da6ed75cb3b72251179dad706c038R285). I'll make a PR ASAP.

**Comment by dadoonet** [comment]
This one will be fixed by your PR #1203.
